### ISSUE TITLE
Support for wrap-around maps

### ISSUE THREAD

**Issue by jfeo** [SFTtech/openage#303]
In games where some teams start along along edge of the map, and some teams starting in the middle, I find that teams in the middle often get squeezed, as they are the closest opponent to all teams on the edges.

If the map wrapped around, so the opposite edges were linked, it would make for interesting, and perhaps more balanced play.

I realize that this is kind of counter to the idea of infinitely generating terrain, but as I imagine that a mode for set-size maps would make sense, in that context wrap-around maps could also be an idea.

**Comment by mic-e** [comment]
This is actually already in the docs

> Spherical/*Toroidal*/Cylindrical surfaces would be possible, too

**Comment by jfeo** [comment]
Ah! I only searched through the issues :-)

**Comment by franciscod** [comment]
it's nice to have it in the issuetracker too :)

**Comment by mic-e** [comment]
Indeed. That way maybe someone will pick it up and implement it. I think it wouldn't actually be that hard; just a few modulo operators somewhere in the chunk code.
